resources/lib/tubecast/chromecast.py

```diff
@@ -42,7 +42,8 @@ class Chromecast(object):
 
     def __init__(self, monitor):
         self._monitor = monitor
-        self._server_thread = Thread(target=self._run_server,
+        self._server_thread = Thread(name="ChromecastServer",
```

Comment:
I gave names to all threads. This made it a lot easier to track down the rogue one and it should hopefully also help in the future.